### PR TITLE
Improvement/dm 2812/updating bulk registration templates y2024

### PR DESCRIPTION
**Pull request by shaikhahme**

### CODE REVIEW

File: content/change-logs/device-management/cumulocity_10-18-540-229_updating-bulk-registration-templates.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - value: change-2c7RdTdXo4
+    label: Improvement
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-4048
+version: 10.18.540.218
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity_10-18-540-229_updating-bulk-registration-templates.md]:
```suggestion
version: 10.18.540.229
```

File: content/change-logs/device-management/cumulocity_10-18-540-229_updating-bulk-registration-templates.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+change_type:
+  - value: change-2c7RdTdXo4
+    label: Improvement
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-4048
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity_10-18-540-229_updating-bulk-registration-templates.md]:
```suggestion
ticket: DM-2812
```

**Review by Oezge-Akin — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/device-management/cumulocity_10-18-540-229_updating-bulk-registration-templates.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-2812
+version: 10.18.540.229
+---
+The wizard for bulk registering LWM2M devices to Cumulocity provides default bulk registration templates.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity_10-18-540-229_updating-bulk-registration-templates.md]:
```suggestion
The wizard for bulk registering LWM2M devices to {{< product-c8y-iot >}} provides default bulk registration templates.
```

**Comment by BeateRixen** [comment]
@shaikhahme I assume, this PR can be merged.

**Pull request merged**